src/content/reference/react/use-client.md

```diff
@@ -215,39 +214,39 @@ This means that the component definition for `FancyText` will both be evaluated
+Client Components can render Server Components because you can pass JSX as props. In this case, `InspirationGenerator` receives `Copyright` as [children](/learn/passing-props-to-a-component#passing-jsx-as-children). However, the `InspirationGenerator` module never directly imports the `Copyright` module nor calls the component, all of that is done by `App`. In fact, the `Copyright` component is fully executed before `InspirationGenerator` starts rendering.
 
 The takeaway is that a parent-child render relationship between components does not guarantee the same render environment.
 
 </DeepDive>
 
 ### When to use `'use client'` {/*when-to-use-use-client*/}
 
-With `'use client'`, you can determine what component usages will be Client Components. As Server Components are default, here is a brief overview of the advantages and limitations to Server Components to determine when you need to mark something as client rendered.
+With `'use client'`, you can determine when components are Client Components. As Server Components are default, here is a brief overview of the advantages and limitations to Server Components to determine when you need to mark something as client rendered.
 
 For simplicity, we talk about Server Components, but the same principles apply to all code in your app that is server run.
 
-#### Advantages {/*advantages*/}
+#### Advantages of Server Components {/*advantages*/}
 * Server Components can reduce the amount of code sent and run by the client. Only client modules are bundled and evaluated by the client.
 * Server Components benefit from running on the server. They can access the local filesystem and may experience low latency for data fetches and network requests.
 
-#### Limitations {/*limitations*/}
+#### Limitations of Server Components {/*limitations*/}
 * Server Components cannot support interaction as event handlers must be registered and triggered by a client.
-	* The browser, as an example client, is responsible for delegating UI clicks to the appropriate `onClick` handler.
+	* For example, event handlers like `onClick` can only be defined in Client Components.
 * Server Components cannot use most Hooks.
-	* When Server Components are rendered, they are conceptually resolved into instructions for the client to interpret and turn into UI primitives. This means that Server Components do not persist in memory after render and do not support re-rendering or state.
+	* When Server Components are rendered, their output is essentially a list of components for the client to render are conceptually resolved into instructions for the client to interpret and turn into UI primitives. Server Components do not persist in memory after render and cannot have their own state.
```

Comment:
I think this needs re-wording after the edit, not sure if this is what you intend

```suggestion
	* When Server Components are rendered, their output is essentially a list of components for the client to render. Server Components do not persist in memory after render and cannot have their own state.
```

Reply:
lol whoops thank you, this is indeed what I meant